interfaces/cython/cantera/reactor.pyx

```diff
@@ -969,10 +969,10 @@ cdef class PressureController(FlowDevice):
         .. deprecated:: 2.5
 
              To be deprecated with version 2.5, and removed thereafter.
-             Replaced by property `pressure_coeff`.
+             Replaced by property 'pressure_coeff'.
```

Comment:
Shouldn't references in docstrings be left as backticks in order to enable linking by Sphinx (see https://cantera.org/documentation/dev/sphinx/html/cython/zerodim.html#cantera.PressureController.set_pressure_coeff)

Reply:
Correct. I meant to replace backticks in warning/error messages (a convention that @bryanwweber pointed out at some point), not here ...

Reply:
The only reason for the convention is that backticks have semantic meaning to Sphinx ("link to something that looks like this string") but on the terminal, the backticks don't have any meaning. Unless you have a terminal font with non-monospace by default and a Markdown parser... I wouldn't put it past someone, I'm sure it exists...

Reply:
That's what I recalled (and why I took the step to replace backticks). As an aside, one of my student is running an installation where non-monospaced error messages do occur.